includes/delete-comments-form.php

```diff
@@ -0,0 +1,122 @@
+<?php
+if( !defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+<div class="wrap">
+<h1><?php _e( 'Delete Comments', 'disable-comments') ?></h1>
+<?php
+
+global $wpdb;
+$comments_count = $wpdb->get_var("SELECT count(comment_id) from $wpdb->comments");
+if ( $comments_count <= 0 ) { ?>
+<p><strong>No comments available for deletion.</strong></p>
+</div>
+<?php
+	exit;
```

Comment:
I see what you're saying, switched to return as the second exit doesn't really work well as an if-else since it's already nested.